content/guides/5.collections.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Collections and Playlists
+slug: collections
+fullpath: /guides/collections
+---
+
+---
+
+# Introduction
+This page describes the similarities and differences between a collection and playlist. Collections and podcasts are another way to organize individual books and podcast episodes (not podcasts).
+
+Both Collections and Playlists:
+- Are Library specific
+- Listening progress is tracked by user (not shared listening state between multiple users)
+- Have a description field
+- Can be reordered
+- Can contain podcast episodes
+- Cannot contain groups of things (other playlists/collections, series, etc)
+- Have a "Play" button at the top which acts as a "Play All" button
+
+Collections:
+- Are Public. Any user with access to the library can see the contents of the collection
+- Can contain books
+
+Playlists:
+- Are Private. Only the user who created the playlist can see it
+
+Screenshots are from an install of v2.2.0 in Docker. The interface is still pretty much the same, but some buttons have moved slightly. These changes are noted in the text.
+
+# Creating a collection or playlist
+When no collections exist, you will be presented with a view similar to this in the book library. The playlist tab does not show up unless you have created a playlist (similar method to creating a collection).
+
+![Initial collection](/guides/collections/1_no_collections.png)
+
+For a book library, a collection or playlist is created from an individual book. You can do this either from the library view or when viewing a specific book's details.
+Collections and playlists are created from book view. To do this, you can either do this from the library view or when viewing a specific book's details. In the second gif, the button has moved to the dropdown list of 3 dots in the same location.
```

Comment:
I think there is some redundancy here?

Reply:
Oh, yep. I didn't notice that with my multiple drafts of sentences getting pasted together. I'll clear that up.